test/faktory_worker/connection/connection_test.exs

```diff
@@ -206,5 +185,20 @@ defmodule FaktoryWorker.Connection.ConnectionTest do
 
       assert reason == "Something went wrong"
     end
+
+    test "should not call not call recv function when sending an END command" do
```

Comment:
typo "not call not call"

Reply:
Good spot, I am going to fix this in #53 to save me having to rebase everything again.